src/hotspot/share/code/SCCache.cpp

```diff
@@ -551,39 +462,31 @@ SCCache::SCCache(const char* cache_path, int fd, uint load_size) {
 
   if (_for_read) {
     // Read cache
-    if (MmapCachedCode) {
```

Comment:
I think you can drop `MmapCachedCode` flag as well.

Reply:
right